### ISSUE TITLE
last two tests always fail no matter what

### ISSUE THREAD

**Issue by stevenbarash** [freeCodeCamp/freeCodeCamp#14279]
Challenge [word-blanks](http://beta.freecodecamp.com/en/challenges/basic-javascript/word-blanks) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```js

function wordBlanks(myNoun, myAdjective, myVerb, myAdverb) {
    var result = "";
    // Your code below this line
    result+= "My "+myAdjective+" "+myNoun+" "+myVerb+" very "+myAdverb+".";

    // Your code above this line
  return result;
}

// Change the words here to test your function
wordBlanks("dog", "big", "ran", "quickly");
```

**Comment by JoolsMcFly** [comment]
Your code works fine for me with Chrome/Firefox Linux.

Are you getting errors in console (F12)?

**Comment by shoesandsocks** [comment]
I was having this problem also -- same "works fine" code as above. Trick I learned from another broken-challenge thread also worked on this one; can't remember which thread it was, but someone said they put a phony param in the URL and reloaded the page. Then pressing Run would cause the test to pass. Something about the param forcing a cache to refresh (or something?)

So I tried that, on this one, and it worked. I just went up to the URL and added something like `?fakeparam` to the end, reload, press Run... ok.
